WooCommerce/UITestsFoundation/Screens/Payments/PaymentsScreen.swift

```diff
@@ -41,6 +61,34 @@ public final class PaymentsScreen: ScreenObject {
         return self
     }
 
+    public func tapCollectPayment() throws -> Self {
+        collectPaymentButton.tap()
+        return self
+    }
+
+    public func enterPaymentAmount() throws -> Self {
+        app.enterText(text: "5")
```

Comment:
I did not start a local review yet, will do that shortly. Just wanted to leave a note that I'd see more flexibility in having the amount passed as an argument, and not have it hardcoded in the screen file.